images/hive/Dockerfile

```diff
@@ -14,18 +14,8 @@ RUN set -x \
 ENV HADOOP_CLASSPATH /opt/hive/hcatalog/share/hcatalog/*:${HADOOP_CLASSPATH}
 ENV HIVE_AUX_JARS_PATH /usr/hdp/current/hive-server2/auxlib
 
-ENV POSTGRESQL_JDBC_JAR postgresql-42.2.2.jar
-# Using mysql-connector-java-8.0.11 resulted in hive schema creation failing due to incorrect syntax.
-ENV MYSQL_JDBC_VERSION mysql-connector-java-5.1.46
-ENV MYSQL_JDBC_JAR $MYSQL_JDBC_VERSION.jar
-
-# Install PostgreSQL JDBC
-RUN set -x \
-    && curl -fSLs -o "$HIVE_HOME/lib/$POSTGRESQL_JDBC_JAR" "https://jdbc.postgresql.org/download/$POSTGRESQL_JDBC_JAR"
-
-# Install MySQL JDBC
-RUN set -x \
-    && curl -fSLs "https://dev.mysql.com/get/Downloads/Connector-J/$MYSQL_JDBC_VERSION.tar.gz" | tar -zx --strip-components=1 -C "$HIVE_HOME/lib" "$MYSQL_JDBC_VERSION/$MYSQL_JDBC_JAR"
+# Install PostgreSQL and MySQL JDBCs
+RUN yum install -y mysql-connector-java postgresql-jdbc && yum clean all && rm -rf /var/cache/yum
```

Comment:
These are definitely not going to necessarily be in the right spots, so you probably need to actually validate this with a mysql/postgresql setup.

Reply:
Eg: "$HIVE_HOME/lib/$POSTGRESQL_JDBC_JAR"

We probably just need to make sure that where these go ends up in our Java classpath, and all these wrapper scripts correctly use that.